src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -250,13 +253,15 @@ private void getRelatedRecords(List<Monument> monuments, String fieldName) {
         );
 
         List<Monument> monumentsWithRecords = this.getWithCriteriaQuery(query);
+        System.out.println("results " + monumentsWithRecords);
 
         String capitalizedFieldName = fieldName.substring(0, 1).toUpperCase() + fieldName.substring(1);
 
         Map<Integer, List> map = new HashMap<>();
         for (Monument monument : monumentsWithRecords) {
             try {
                 map.put(monument.getId(), (List) Monument.class.getDeclaredMethod("get" + capitalizedFieldName).invoke(monument));
+                System.out.println("put " + monument.getId() + " " + map.get(monument.getId()));
             } catch (NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
                 System.err.println("Invalid field name: " + fieldName);
                 System.err.println("Occurred while trying to use getter: get" + capitalizedFieldName);
```

Comment:
Would it be such a bad thing to have these methods inside the new Repository classes instead?

Reply:
You can't since they're interfaces

Reply:
Oh yeah forgot about that. Kind of a bummer.